pkg/bpftrace/Dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM linuxkit/alpine:ace75d0ec6978762d445083d6c8c6336bdb658ed AS build
+RUN apk add --update \
+    bison \
+    build-base \
+    clang-dev \
+    clang-static \
+    cmake \
+    elfutils-dev \
+    flex-dev \
+    git \
+    linux-headers \
```

Comment:
This compiles it against the headers from Alpine Linux but not the LinuxKit kernels. Is that a problem?

Reply:
Hm – not sure about that. I seem to think this will be fine as I was able to run the compiled packaged but is there any other way I could figure that out?